doc/tools/docgram/fullGrammar

```diff
@@ -2447,6 +2497,7 @@ or_and_intropattern_loc: [
 as_or_and_ipat: [
 | "as" or_and_intropattern_loc
 |
+| "as" or_and_intropattern
```

Comment:
That's a surprising difference. With the current duplication of code, there is always a risk of divergence between the two grammars.